en/tools/config-catalog-generator/data/apim.token.revocation.toml

```diff
@@ -1,5 +1,5 @@
 [apim.token.revocation]
-notifier_impl = "org.wso2.carbon.apimgt.keymgt.events.TokenRevocationNotifierImpl"
+notifier_impl = "org.wso2.carbon.apimgt.notification.TokenRevocationNotifier"
```

Comment:
The class needs to be `org.wso2.carbon.apimgt.notification.TokenRevocationNotifierImpl` as per the default.json.